octopoes/octopoes/models/ooi/geography.py

```diff
@@ -0,0 +1,17 @@
+from typing import Literal
+
+from octopoes.models import OOI, Reference
+from octopoes.models.persistence import ReferenceField
+
+
+class GeographicPoint(OOI):
+    object_type: Literal["GeographicPoint"] = "GeographicPoint"
+
+    ooi: Reference = ReferenceField(OOI)
+
+    longitude: float
+    latitude: float
```

Comment:
```suggestion
    longitude: confloat(ge=-180.0, le=180.0)
    latitude: confloat(ge=-90.0, le=90.0)
```

Reply:
The `con*` are discouraged in favor of using `Annotated` with `Field` instead. See Pydantic docs about constrained types. But should look like `longitude: Annotated[float, Field(strict=True, ge=-180.0, le=180.0)]`.

Ideally we could use Pydantic extra types, which includes a `Longitude` and `Latitude` data types and some other possible useful types. This however involves an extra package, which we should consider carefully before using it. In any cases if we wish to use that, it can still be implemented later without backwards compatibility.